addons/medical_engine/XEH_preInit.sqf

```diff
@@ -6,6 +6,19 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+// Define "Constants" variables
+HEAD_DAMAGE_THRESHOLD = HEAD_DAMAGE_THRESHOLD_DEFAULT;
```

Comment:
Maybe also note that both sides are macros. I know its clear from the allcaps.
But a beginner will just think that HEAD_DAMAGE_THESHOLD is a variable.

Reply:
I think we should just replace all use of `HEAD_DAMAGE_THRESHOLD` with it's var

Reply:
on second thought, I think it might be cleaner to just keep the macros